lib/graphql_devise/rails/routes.rb

```diff
@@ -39,9 +39,9 @@ def mount_graphql_devise_for(resource, opts = {})
         skip:   [:sessions, :registrations, :passwords, :confirmations, :omniauth_callbacks, :unlocks]
       )
 
-      authenticable_type = opts[:authenticable_type] ||
+      authenticatable_type = opts[:authenticatable_type] ||
                            "Types::#{resource}Type".safe_constantize ||
-                           GraphqlDevise::Types::AuthenticableType
+                           GraphqlDevise::Types::AuthenticatableType
```

Comment:
Please align this and the line above with `opts`

Reply:
Got it. Not sure if this conforms to your styling preferences but what about this:
```ruby
      authenticatable_type =
        opts[:authenticatable_type] ||
        "Types::#{resource}Type".safe_constantize ||
        GraphqlDevise::Types::AuthenticatableType
```

Reply:
Our current rubocop config prefers the other approach

Reply:
Got it. I'll make the proper adjustment and commit the change.